corehq/apps/reports/models.py

```diff
@@ -1097,3 +1099,21 @@ def remove_custom_export(cls, domain, export_id):
     def clear_caches(self):
         super(HQGroupExportConfiguration, self).clear_caches()
         self.by_domain.clear(self.__class__, self.domain)
+
+
+class ReportsSidebarOrdering(models.Model):
+    domain = models.CharField(
+        max_length=256,
+        null=False,
+        blank=False,
+        unique=True
+    )
+    config = JSONField(
+        default=list,
+        help_text=(
+            "An array of arrays. Each array represents a heading in the sidebar navigation. "
+            "The first item in each array is a string, which will be the title of the heading. The second item in "
+            "the array is another array, each item of which is the name of a report class. Each of these reports "
+            "will be listed under the given heading in the sidebar nav."
```

Comment:
I had an example here to begin with, but the admin app renders the help text in `<p>` tags so I can't nice formatting for the json without mucking with the admin app a little, which didn't seem worth it. What do you think of putting an example in a comment here instead of adding it to the help_text?